public/components/ContentPanel/ContentPanel.tsx

```diff
@@ -38,14 +38,16 @@ type ContentPanelProps = {
   titleContainerStyles?: React.CSSProperties;
   actions?: React.ReactNode | React.ReactNode[];
   children: React.ReactNode | React.ReactNode[];
+  contentPanelClassName?: string;
 };
 
 const ContentPanel = (props: ContentPanelProps) => (
   <EuiPanel
     style={{ paddingLeft: '0px', paddingRight: '0px', ...props.panelStyles }}
+    className={props.contentPanelClassName}
   >
     <EuiFlexGroup
-      style={{ padding: '0px 10px', ...props.titleContainerStyles }}
+      style={{ padding: '0px 20px', ...props.titleContainerStyles }}
```

Comment:
How about we change this padding as 0
```
style={{ padding: '0px', ...props.titleContainerStyles }}
```

And set the padding of `EuiPanel` as `20px`, so we can align title and content body.
```
<EuiPanel
    style={{ padding: '20px', ...props.panelStyles }}
    className={props.contentPanelClassName}
  >
```

Reply:
Changing the padding of `EuiPanel` to 20px will make the `EuiHorizontalRule` looks  disconnected to the Panel. I changed the style for Live chart, and below is the screenshot. 
![Screen Shot 2020-04-27 at 5 52 28 PM](https://user-images.githubusercontent.com/59710443/80434923-f6d48a80-88af-11ea-90cc-e5486b508175.png)

This is not same as the UX mock up.

Reply:
Sure, let's follow the UX mock up